### PR TITLE
chore : NR1641 - whats included -Dashboards, Alerts & DataSources section

### PR DESCRIPTION
**Pull request by dikshab0202**
JIRA : https://issues.newrelic.com/browse/NR-1641
Description: This PR has only whats included section, for which created 3 separate new components for each section and 4 components for 4 new icons.
Note: "How to use quickstart" section is in another PR https://github.com/newrelic/instant-observability-website/pull/142

image

image

image

image

### CODE REVIEW

File: src/components/QuickstartDashboards.js [hunk 1]
```diff
@@ -19,18 +62,25 @@ const QuickstartDashboards = ({ quickstart }) => (
...
-            margin-bottom: 1rem;
-          }
-        `}
-      >
-        <h3>{dashboard.name}</h3>
-        {dashboard.description && <p>{dashboard.description}</p>}
-        <ImageSlider height={400} images={dashboard.screenshots} />
+      <div key={dashboard.name}>
+        <div>
+          <Slider {...settings}>
```
Comment (aswanson-nr) [inline src/components/QuickstartDashboards.js:67]: I noticed you're using a slider here, but in the preview it does not appear to be displaying them as a slider.
Screen Shot 2022-04-26 at 11 41 01 AM

File: src/components/WhatsIncluded/Dashboards.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                {quickstart.dashboards?.length > 0 ? (
+                    <QuickstartDashboards quickstart={quickstart} />
+                ) : (
+
+                    <p>
+                        This quickstart doesn't include any dashboards. Do you think it should?
+                        <br />
+                        You can edit this quickstart to add helpful components. View the
+                        repository and open a pull request.
+                    </p>
```
Comment (aswanson-nr) [inline src/components/WhatsIncluded/Dashboards.js:53]: The other `WhatsIncluded` components use the `EmptyState` component for this case. I think either is fine, but we should be consistent across these components.

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Overall looks fantastic! Just noticed some odd behavior in a few spots, I attached some screenshots to help with debugging.
Screen Shot 2022-04-26 at 11 35 36 AM

File: src/components/QuickstartDashboards.js (hunk 1 above)
Comment (dikshab0202) [reply src/components/QuickstartDashboards.js:69]: Hi Alec, it is resolved. Can you please review once again?

File: src/components/WhatsIncluded/Dashboards.js (hunk 2 above)
Comment (dikshab0202) [reply src/components/WhatsIncluded/Dashboards.js]: Replaced this code.

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**